cookbooks/bcpc/recipes/serial_console.rb

```diff
@@ -0,0 +1,59 @@
+def numeric?(data)
+        Float(data) != nil rescue false
+end
+
+serial_devices = {}
+# we are not interested in USB or any other serial at the moment
+serial_dev_type = "ttyS"
+serial_driver_info = "/proc/tty/driver/serial"
+File.open(serial_driver_info, "r") do |serials|
```

Comment:
Hi Lev, could you add a comment showing the expected format you expect from the serial file? (I don't suspect it will change but if it does, it will help some future code spelunker understand your logic for a fix.)